### DISCUSSION TITLE
1 click upgrade Module is not recognised as being already updated in its latest version:

### DISCUSSION THREAD

**Discussion by VertNature** [community/community#102917] [New to GitHub]
Upgrade ( from 1.6.1.18 to 1.6.1.24) is blocked as a consequence of this message: 

 "Le module de mise à jour en 1 clic est à jour (votre version est v4.14.2) "  Mettre à jour    Téléchargement    X
![Capture web_19-1-2024_13209_vertnature fr](https://github.com/community/community/assets/50493174/1473600f-e5db-4ea2-9343-713b785474de)

Any attempt to upgrade the module logically aborts as it is already updated.

I need help please !

**Comment by queenofcorgis** [comment]
Hi @VertNature ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this piece of software. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
